src/printer/index.js

```diff
@@ -128,6 +128,29 @@ function printForIn(path, print) {
   ]);
 }
 
+function printWith(path, items, print) {
+  return concat([
+    group(concat(["with", line, join(", ", items), ":"])),
+    indent(concat([hardline, printBody(path, print)]))
+  ]);
+}
+
+function printPython2With(path, items, print) {
+  items.push(printWithItem(path, print));
+  let result;
+
+  const n = path.getValue();
+  if (n.body.length === 1 && n.body[0].ast_type === "With") {
+    path.each(p => {
```

Comment:
is this idiomatic? i couldn't find a better way but i might not have looked hard enough

the idea is that `body` is an array, but in this case it's of length 1, so we want to call something on it

Reply:
Maybe:

```js
return concat(
  path.map(
    p => printPython2With(p, items, print),
    "body"
  )
)
```

Reply:
Oops. Now I feel embarrassed for not thinking of that. Thanks!